src/main/java/school/hei/haapi/authentication/JwtConfiguration.java

```diff
@@ -0,0 +1,56 @@
+package school.hei.haapi.authentication;
+
+import java.io.Serializable;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtConfiguration implements Serializable {
+  private static final String USER_POOL_ID = ("us-east-1_AvlIh6liI");
+  private static final String REGION = ("us-east-1");
+  private static final String HTTP_HEADER = "Authorization";
+  private static final int CONNECTION_TIMEOUT = 2000;
```

Comment:
Make it configurable

Reply:
Do we specify the value of the userPoolId  in the `application-profile.properties` or set the variable manually in the environment before and specify its name instead ?